web/pages/notifications.tsx

```diff
@@ -865,6 +859,8 @@ function getReasonForShowingNotification(
         )
       )
         reasonText = `resolved`
+      else if (sourceUpdateType === 'closed')
+        reasonText = `please resolve your question`
```

Comment:
I'm not that sure about this. Maybe it's fine.

Reply:
Have any ideas for alternatives?

Reply:
we can always change it later!